paasta_tools/cli/cmds/logs.py

```diff
@@ -100,9 +103,37 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+
+    status_parser.add_argument(
+        '-a', '--ago', dest='time_ago',
+        help='How long ago to get logs from, uses friendly durations like 32m or 1w. Defaults to 30 minutes',
+    )
+    status_parser.add_argument(
+        '-t', '--time', dest='time',
```

Comment:
That makes sense, I'll change it to --to and --from with the option of both timestamps and human readable time deltas.